packages/modules-sdk/src/loaders/module-loader.ts

```diff
@@ -22,7 +22,7 @@ const registerModule = async (
   configModule: ConfigModule,
   logger: Logger
 ): Promise<{ error?: Error } | void> => {
-  const constainerName = resolution.definition.registrationName
+  const containerName = resolution.definition.registrationName
```

Comment:
I'm correcting this typo in another PR and this file is changing quite a bit, so I believe it is better to revert these changes to avoid conflict.

Reply:
reverted in favour of that PR